Commander: add COM_WIND_MAX_ACT param

sfuhrer opened the pull request:
### Solved Problem
Currently the action after a "high wind speed detected" is hard coded to RTL. We would like to expose it to the user to easily switch on/off this failsafe, and doing that over the currently only interface COM_WIN_MAX (eg set to -1 to disable) is not as easy as setting a boolean. 
For some use cases, setting the action to Land instead of RTL is more appropriate, and even further options like Terminate could be added.

### Solution
Add new param `COM_WIND_MAX_ACT` with fields:
- None (in that case COM_WIND_MAX is not looked at at all)
- Warning ( in that case, if the wind is above COM_WIND_MAX, it publishes a warning that replaces the one for COM_WIND_WARN)
- Hold
- Return
- Terminate
- Land

<del>The issue I have with this way of implementing it is that it is not possible to disable the failsafe action after it already triggered by changing `COM_WIND_MAX_ACT`. For that we would need an action reset - or is that already possible @bkueng ?
Otherwise we could take `COM_WIND_MAX_ACT` already into account for the setting of `wind_limit_exceeded`. it's then though no longer nicely separated. 

I further wonder if we can combine `COM_WIND_WARN` and `COM_WIND_MAX` into one param and start warning at a fixed percentage of `COM_WIND_MAX`, eg 75%. For most applications that would be good enough, but for others this customization is probably required.

bkueng commented:
> For that we would need an action reset - or is that already possible @bkueng ?
Otherwise we could take COM_WIND_MAX_ACT already into account for the setting of wind_limit_exceeded. it's then though no longer nicely separated.

This should already be the case: https://github.com/PX4/PX4-Autopilot/blob/f498b90c4149548ab1a108ceebb9e3d9dead5d22/src/modules/commander/failsafe/framework.cpp#L306

bkueng commented on src/modules/commander/failsafe/failsafe.h:
```diff
@@ -123,6 +123,12 @@ class Failsafe : public FailsafeBase
 		Hold_mode = 2,
 	};
 
+	enum class command_after_high_wind_failsafe : int32_t {
```
Ideally I'd like to keep these generic and reuse the same enum for most params where it makes sense.

sfuhrer replied:
Okay I've align it now with the GF failsafe settings. Does that make sense to you? Or which one would you have re-used?

bkueng replied:
Yes makes sense. I would also make the enum name generic and re-use it.

sfuhrer commented:
@bkueng changed the implementation, rebased and updated the PR description. Please have another look, from SITL testing it now does what was required.

bkueng reviewed (approved):
Looks good, I leave it up to you if you want to make the enum more generic.

sfuhrer commented:
> Looks good, I leave it up to you if you want to make the enum more generic.

I don't want to hold off merging that one. There will be a lot of discussion around whether we can have the failsafe options generic for all failure cases.
